app/javascript/packs/hello_erb.js.erb

```diff
@@ -0,0 +1,6 @@
+// Run this example by adding <%%= javascript_pack_tag 'hello_erb' %> to the head of your layout file,
```

Comment:
@cdmo can we delete this file?

Reply:
yeah, and in fact a lot of this PR can be deleted, it's a total mess! 🙃